recipes/ipydatetime/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "ipydatetime" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9b083e08efef140c29091fe114ea2921f88989e56f0b5eecdde9ac0157bb0b10
+
+build:
+  noarch: python
+  number: 0
+  script:
+    - {{ PYTHON }} -m pip install . -vv
+    - {{ PYTHON }} -c '__import__("os").unlink("pyproject.toml")'
```

Comment:
invert the order here

Reply:
gotcha; sorry I missed that

Reply:
Just curios, why keeping the pyproject.toml did not work here?

Reply:
@bollwyvl will have to answer that; I was just blindly doing everything he was telling me to do